scripts/report-bundle-size.js

```diff
@@ -26,14 +34,16 @@ const main = () => {
                     bundles.forEach((bundle) => {
                         const metric = `mobify_platform_sdks.bundle_size_byte`
                         const value = bundle.size
-                        childProc.spawnSync('dog', [
+                        // TODO: is dog installed yet?
+                        const {status, stderr, error} = childProc.spawnSync('dog', [
```

Comment:
Since [last October](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2070), our datadog has failed to run properly because it has not been installed when this command runs.